test/server/cards/09.1-FtE/FalseLoyalities.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('False Loyalites', function() {
+    integration(function() {
+        describe('False Loyalites\'s ability', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['matsu-berserker','shinjo-shono'],
+                        honor: 10
+                    },
+                    player2: {
+                        inPlay: ['venerable-historian'],
+                        hand: ['false-loyalties'],
+                        honor: 8
+                    }
+                });
+
+                this.berserker = this.player1.findCardByName('matsu-berserker');
+                this.shono = this.player1.findCardByName('shinjo-shono');
+
+                this.historian = this.player2.findCardByName('venerable-historian');
+
+                this.noMoreActions();
+            });
+
+            it('should trigger after a opponent wins a conflict', function() {
+                this.initiateConflict({
+                    type: 'political',
+                    attackers: [this.shono],
+                    defenders: []
+                });
+                this.noMoreActions();
+                expect(this.player2).toHavePrompt('Any Reactions');
+                expect(this.player2).toBeAbleToSelect('false-loyalities');
+            });
```

Comment:
Yeah it isn't triggering because Matsu Berserker has a dash in political so is ineligble to be moved to the conflict so the targeting requirement fails.
BTW, there's a console log you can uncomment in `triggeredability.js` that can sometimes give you a clue as to why something is not triggering.

Reply:
thanks I fixed those bugs and added more test and the card seems to be working now